pkg/controllers/uiplugin/plugin_info_builder.go

```diff
@@ -109,7 +111,9 @@ func PluginInfoBuilder(plugin *uiv1alpha1.UIPlugin, pluginConf UIPluginsConfigur
 		}
 	case uiv1alpha1.TypeTroubleshootingPanel:
 		{
-			return createTroubleshootingPanelPluginInfo(plugin, namespace, plugin.Name, image, []string{})
+			pluginInfo, _ := createTroubleshootingPanelPluginInfo(plugin, namespace, plugin.Name, image, []string{})
```

Comment:
The error should not probably be ignored.

Reply:
Done